src/conf/local-file.yaml

```diff
@@ -2,7 +2,7 @@ application:
   name: knoxautopolicy
   network:
     operation-mode: 1                             # 1: cronjob | 2: one-time-job
-    operation-trigger: 5
+    operation-trigger: 1
```

Comment:
Operation trigger of 1 or 5 is too low

Reply:
@nyrahul In case of neteork opertaion for discoevry-engine, we are not processing all logs. We are processing only bare minimum logs which contains TCP and UDP related data. In that case, we need a minimum trigger of 5 to start the processing.